libraries/freertos_plus/standard/freertos_plus_tcp/test/aws_freertos_tcp_test_access_declare.h

```diff
@@ -1,6 +1,6 @@
 /*
- * Amazon FreeRTOS
- * Copyright (C) 2018 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ * FreeRTOS+TCP V2.0.11
+ * Copyright (C) 2017 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
Aren't these new files and need to be year 2019?

Reply:
Updated. Thanks for catching that.